docs/user-guide.md

```diff
@@ -0,0 +1,224 @@
+
+#### `appliance-config.yaml` Example:
+```yaml
+apiVersion: v1beta1
+kind: ApplianceConfig
+ocpRelease:
+  version: 4.12.10
+  channel: stable
+  cpuArchitecture: x86_64
+diskSizeGB: 200
+pullSecret: <redacted>
+sshKey: <redacted>
+userCorePass: <redacted>
+```
+
+### Build the image
+#### Demo
+[![asciicast](https://asciinema.org/a/591871.svg)](https://asciinema.org/a/591871)
+
+* Make sure you have enough free disk space.
+* Building the image may take several minutes.
+* The option `--privileged` is used because the `openshift-appliance` container needs to use `guestfish` to build the image.
+* The option `--net=host` is used because the `openshift-appliance` container needs to use the host networking for the image registry container it runs as a part of the build process .
+```shell
+podman run --rm -it --privileged --net=host -v $APPLIANCE_ASSETS:/assets:Z $APPLIANCE_IMAGE build
+
+```
+Result
+```shell
+INFO Successfully downloaded CoreOS ISO
+INFO Successfully generated recovery CoreOS ISO
+INFO Successfully pulled container registry image
+INFO Successfully pulled OpenShift 4.12.10 release images required for bootstrap
+INFO Successfully pulled OpenShift 4.12.10 release images required for installation
+INFO Successfully generated data ISO
+INFO Successfully downloaded appliance base disk image
+INFO Successfully extracted appliance base disk image
+INFO Successfully generated appliance disk image
+INFO Time elapsed: 8m0s
+INFO
+INFO Appliance successfully created in assets directory: assets/appliance.raw
+INFO
+INFO Create configuration ISO using: openshift-install agent create config-image
+INFO Download openshift-install from: https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.12.10/openshift-install-linux.tar.gz
+```
+******
+
+
+## Copy Appliance Image to Disk - Factory
+* Baremetal servers: use `dd`
+* Virtual Machines: configure the disk to use `/path/to/appliance.raw`
+
+## OpenShift Cluster Install - User Site
+### Demo 
+  [![asciicast](https://asciinema.org/a/590824.svg)](https://asciinema.org/a/590824)
+
+### Download `openshift-install`
+* So far, the generated image has been completely generic. To install the cluster, the installer will need cluster-specific configuration.
+* The `openshift-install` version mentioned above does not include the config-image API. This API is available starting from version `4.14`.
+* * Download and extract the latest `openshift-install` version from: https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/latest/openshift-install-linux.tar.gz
+```shell
+  curl -O https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/latest/openshift-install-linux.tar.gz
+  tar -xvf openshift-install-linux.tar.gz
+  ```
+
+### Generate a Cluster Configuration Image
+* Create a configuration directory
+  
+**:warning: for CLUSTER_CONFIG, use an absolute path.**
```

Comment:
we can have the exact same warning as in line 52:
'**:warning: use absolute paths.**'

Reply:
Done.